### PR TITLE
New version: Appest.Dida version 4.4.1.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | 滴答清单 | 滴答清单 版本 4.4.1.0    |
| Version     | 4.4.1.0     | 4.4.1.0 |
| Publisher   | 杭州随笔记网络技术有限公司   | 杭州随笔记网络技术有限公司      |
| ProductCode |           | {1A434D02-8C9A-41A2-9BBE-C97A1E31ABC1}_is1    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95408)

### CODE REVIEW

File: manifests/a/Appest/Dida/4.4.1.0/Appest.Dida.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- ticktick
+ReleaseDate: 2023-01-03
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cdn.dida365.cn/download/win/dida_win_setup_release_x86_4410.exe
+  InstallerSha256: 5E01556B4F0EA6D0C6D3BB96DEA064DCBF2E5FAD87F6BB1D1C71DB9120EB9761
```
Comment (rodrigoborgesdeoliveira) [inline manifests/a/Appest/Dida/4.4.1.0/Appest.Dida.installer.yaml:15]:
```suggestion
  InstallerSha256: 0BD707696BAB9572CA606F91170494BAF5499C3BE8509DC914299DDDC7D1D03E
```

File: manifests/a/Appest/Dida/4.4.1.0/Appest.Dida.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Architecture: x64
+  InstallerUrl: https://cdn.dida365.cn/download/win64/dida_win_setup_release_x64_4410.exe
+  InstallerSha256: A4175507F2D3AD73C8A9AF84AEF9C4AB515E3718FC201B9245D6FA21535F889E
```
Comment (rodrigoborgesdeoliveira) [inline manifests/a/Appest/Dida/4.4.1.0/Appest.Dida.installer.yaml:18]:
```suggestion
  InstallerSha256: 116AAD8AC3CCB8289FA7884C7C18E7892B1F7AE3401401DFC5A5CC4F6305963B
```

**Review by rodrigoborgesdeoliveira — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Merge conflicts;

**Pull request closed without merging**